common/swagger/swagger.yaml

```diff
@@ -203,6 +208,27 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Row'
+      inputShapleyContributions:
```

Comment:
personally I would expect something similar to the output of `zip(dataframe_preds, dataframe_shapely)` in python... sorry if syntax is off here. Just one table.

Reply:
as far as I can tell, shapley results will always have `contrib_` prefix. So there wouldn't be danger of having collision with predictions or inputs.

Reply:
I'd say that would complicate things. Models have given schema. If you add additional columns, validation may fail. Or shaps don't need to make it to the output when the output is constructed from the schema.

Reply:
Resolving this comment as combining the prediction and shapley dataframes might cause issues down the pipeline